src/dataCorrectionsTool/incentiveEligibilityTool.js

```diff
@@ -0,0 +1,411 @@
+                        handleParticipantPaymentTextContent(participantData, isEligibleForIncentiveUpdate);
+                        dateOfEligibilityInput.disabled = true;    
+                    }
+                    setupModalContent(participantPaymentRound);
+                    dateOfEligibilityInput.textContent = setIncentiveEligibleInputDefaultValue();
+                } catch (error) { 
+                    console.error("Failed to check if participant is already incentive eligible: ", error);
+                }
+                
+
+            } else { 
+                toggleSubmitButton();
+                participantPaymentRound = null;
+                isEligibleForIncentiveUpdate = null;
+                selectButton.textContent = e.target.textContent;
+                setIncentiveEligibleInputDefaultValue();
+                isIncentiveEligibleNote.innerHTML = ``;
+                dateOfEligibilityInput.disabled = false;
+                handleParticipantPaymentTextContent(participant, isEligibleForIncentiveUpdate);
+            }
+        });
+    }
+};
+
+const handleParticipantPaymentTextContent = (participant, isEligibleForIncentiveUpdate) => { 
+    const incentiveStatusText = document.getElementById('incentiveStatusText');
+    const isIncentiveEligibleNote = document.getElementById('isIncentiveEligibleNote');
+    const dateOfEligibilityInput = document.getElementById('dateOfEligibility');
+    const dateOfEligibilityText = document.getElementById('dateOfEligibilityText');
+    if (!incentiveStatusText || !isIncentiveEligibleNote || !dateOfEligibilityInput || !dateOfEligibilityText) return;
+    
+    const { paymentRound, baseline, baselinePaymentDate } = fieldMapping; 
+
+    if (isEligibleForIncentiveUpdate) {
+        incentiveStatusText.textContent = 'Incentive Eligible Status: Yes';
+        dateOfEligibilityText.textContent = 'Date of Eligibility: N/A';
+
+    } else if (isEligibleForIncentiveUpdate === false) {
+        incentiveStatusText.textContent = 'Incentive Eligible Status: No';
+        dateOfEligibilityText.textContent = `Date of Eligibility: ${humanReadableMDYTimeZoneOffset(participant?.[paymentRound]?.[baseline]?.[baselinePaymentDate])}`; // TODO: Add flexibility for other payment rounds
+        isIncentiveEligibleNote.innerHTML = `<span><i class="fas fa-check-square fa-lg" style="color: #4CAF50; background: white;"></i> This participant is already incentive eligible.</span>`;
+    } else {
+        incentiveStatusText.textContent = 'Incentive Eligible Status: ';
+        dateOfEligibilityText.textContent = 'Date of Eligibility:';
+        dateOfEligibilityInput.textContent = setIncentiveEligibleInputDefaultValue();
+    }
+
+}
+    
+
+const setIncentiveEligibleInputDefaultValue = () => { 
+    const dateOfEligibilityInput = document.getElementById('dateOfEligibility');
+    if (dateOfEligibilityInput) {
+        const today = new Date();
+        const year = today.getFullYear();
```

Comment:
`new Date().toLocaleDateString("en-CA", {timeZone:"America/New_York"})` should do it.